ntbs-service/Pages/Shared/_NotificationLayout.cshtml

```diff
@@ -104,7 +104,7 @@
     <h2 class="govuk-heading-s" id="manage-notification-title">
         Manage Notification
     </h2>
-    @if (Model.Notification.NotificationStatus == NotificationStatus.Draft)
+    @if (Model.Notification.NotificationStatus == NotificationStatus.Draft && Model.HasEditPermission)
     {
         <button nhs-button-type="Standard" name="actionName" value="Submit">Submit</button>
     }

```

Comment:
Think you referenced it in another 'discussion', but just thought I would tag it here as denotify should probably also have an edit check on it